scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -13,8 +13,7 @@ workflow GvsExtractCallset {
         File reference_index
         File reference_dict
 
-        String fq_samples_to_extract_table
-        String fq_cohort_extract_table
+        String fq_cohort_extract_table_prefix
```

Comment:
hm, I'm wondering if the previous way of providing the fq_tables explicitly is more flexible since it ties the implementation a little less to PrepareCallset. Now, the `fq_sample_to_extract_table` and `fq_cohort_extract_table` would need to be part of the same dataset even though they technically don't have to be.

That said, this works just fine for AoU. Just thinking about some other use cases.